doc/changes/changes_0.3.0.md

```diff
@@ -0,0 +1,9 @@
+# Virtual Schema Common Document 0.3.0, released 2020-09-04
+
+## Summary
+
+Fixed bug #12
```

Comment:
If it's a bugfix, you should change the minor version number

Reply:
And I think you don't need a summary here, it's redundant

Reply:
Well, it is a bug, but it's solution did change the api for the dynamodb-vs. Hence I changed a major version number. Should I only increase the minor version number in that case anyway?